css/_darkvariables.scss

```diff
@@ -2,7 +2,7 @@ $blue-500: #0a84d6; /*light to dark in dark theme*/
 $blue-300: #136fac;
 $blue-100: #1d5a83;
 
-$orange-400: #ffc285;
+$orange-400: #c27c36;
 $orange-500: #FFA726;
```

Comment:
maybe we should change these aswell? im not sure we use them but at some point we might